base/changes.txt

```diff
@@ -1,3 +1,21 @@
+2025-03-21  Frank Mittelbach  <Frank.Mittelbach@latex-project.org>
+
+	* ltoutput.dtx
+	(subsection{The output routine configuration components}):
+	Back up by \dp\@outputbox in the footnotes-floats-legacy plug if there
+	are no footnotes (footmisc/19)
+
+	(subsection{The output routine configuration components}):
+	Add experimantal socket for baseline attachment
```

Comment:
experimantal -> experimental

Reply:
Also, "Add an . . . "

Reply:
@car222222, if you add a comment/suggestion to a comment by somebody else whihc I have already handled and marked as "resolved" it helps a lot of you then also "unresolve" it, because then it is much easier to find.

Vice versa, if you are ok with a change made to an earlier comment from you, makring it "resolved" helps me to not spend a lot of time.

(not sure your interface has these buttons, but if it has ...) thanks